src/hooks/AllPaymentHooks.res

```diff
@@ -418,6 +421,92 @@ let useRedirectHook = () => {
             },
           )
         }
+      | "third_party_sdk_session_token" => {
+          // TODO: add event loggers for analytics
+          let session_token = Option.getOr(nextAction, defaultNextAction).session_token
+          let openProps = {
+            PlaidTypes.onSuccess: success => {
+              retrievePayment(Payment, clientSecret, publishableKey, ~isForceSync=true)
```

Comment:
Post completion of this pis flow using plaid, why we are doing a retrieve using forceSync=true?

Reply:
To sync payment-status between hyperswitch and plaid.
On Plaid flow completion, hyperswitch backend have no info about the payment status (success/ failure/ processing), we do a sync call to get the status synced on hs and plaid.

We did the same thing in web-sdk as well. [PlaidSDKIframe.res: callbackOnSuccessOfPlaidPaymentsFlow](https://github.com/juspay/hyperswitch-web/blob/827ae3a1047f1a463fa03b4e62ada651fed531a8/src/Payments/PlaidSDKIframe.res#L52)